Phase-5/SearchEngine/Classes/FileReader.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using SearchEngine.Interfaces;
+
+namespace SearchEngine
```

Comment:
namespace SearchEngine.Classes
the same thing for other files in the "Classes" directory